Audit service fails with Oracle DB

Audit/query service fails to start while using Oracle DB (7.1.0-M12):
```
Caused by: liquibase.exception.DatabaseException: ORA-00969: missing ON keyword
activiti-cloud-query       |  [Failed SQL: (969) CREATE INDEX IF NOT EXISTS audit_event_event_id_idx ON audit_event(event_id)]
activiti-cloud-query       |    at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:430)
activiti-cloud-query       |    at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:87)
activiti-cloud-query       |    at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:159)
activiti-cloud-query       |    at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1276)
activiti-cloud-query       |    at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1258)
activiti-cloud-query       |    at liquibase.changelog.ChangeSet.execute(ChangeSet.java:622)
activiti-cloud-query       |    ... 193 common frames omitted
activiti-cloud-query       | Caused by: java.sql.SQLSyntaxErrorException: ORA-00969: missing ON keyword
```

--- a/activiti-cloud-audit-service/activiti-cloud-services-audit-liquibase/src/main/resources/config/audit/liquibase/changelog/05-alter.oracle.schema.m11.sql
+++ b/activiti-cloud-audit-service/activiti-cloud-services-audit-liquibase/src/main/resources/config/audit/liquibase/changelog/05-alter.oracle.schema.m11.sql
@@ -1,1 +1,1 @@
-CREATE INDEX IF NOT EXISTS audit_event_event_id_idx ON audit_event(event_id);
+CREATE INDEX audit_event_event_id_idx ON audit_event(event_id);

--- a/activiti-cloud-audit-service/activiti-cloud-services-audit-liquibase/src/main/resources/config/audit/liquibase/master.xml
+++ b/activiti-cloud-audit-service/activiti-cloud-services-audit-liquibase/src/main/resources/config/audit/liquibase/master.xml
@@ -64,6 +64,11 @@
   </changeSet>
 
   <changeSet author="activiti-audit" id="alter5-oracle-schema-m11" dbms="oracle">
+    <preConditions onFail="CONTINUE">
+      <not>
+        <indexExists indexName="audit_event_event_id_idx"/>
+      </not>
+    </preConditions>  
     <sqlFile dbms="oracle"
              encoding="utf8"
              path="changelog/05-alter.oracle.schema.m11.sql"
@@ -71,7 +76,6 @@
              splitStatements="true"
              stripComments="true"/>
   </changeSet>
-
 
   <changeSet author="activiti-audit"
              id="initial-schema-m3" dbms="postgresql">

